### ISSUE TITLE
VS Code porting

### ISSUE THREAD

**Issue by kflore16** [madskristensen/DialToolsForVS#40]
### Installed product versions
- Visual Studio: 2019

### Description
I'm not quite sure if this has ever been mentioned nor did my search find any results but would there by any possibilities of adding this amazing extension for VS Code?

**Comment by petero-dk** [comment]
There is a sample port now here: https://github.com/petero-dk/DialToolsForVSCode

**Comment by madskristensen** [comment]
I can't port it but happy to see you got it started :)

**Comment by petero-dk** [comment]
Yes, it is obviously a complete rewrite and uses some hacks to actually function. It was more for fun that anything else :-)
